core/bessctl.cc

```diff
@@ -937,6 +963,9 @@ class BESSControlImpl final : public BESSControl::Service {
   Status GetModuleInfo(ClientContext* context,
                        const GetModuleInfoRequest& request,
                        GetModuleInfoResponse* response) {
+    if (is_any_worker_running()) {
```

Comment:
get_module_info() does not require worker threads to be stopped (see sn_handlers() in snctl.cc)